.github/gesttalt.yml

```
@@ -0,0 +1,194 @@
+# Gesttalt Platform Configuration
+# This file configures the Gesttalt blogging platform settings
+
+platform:
+  name: Gesttalt
+  version: 0.1.0
+  description: A free-software blogging platform for developers
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)